src/util/stream_utils.js

```diff
@@ -30,20 +29,28 @@ function get_tap_stream(func) {
 }
 
 const async_pipeline = util.promisify(stream.pipeline);
-async function pipeline(streams) {
+async function pipeline(streams, reuse_last_stream) {
     if (!streams || !streams.length) throw new Error('Pipeline called without streams');
     if (streams.find(strm => strm.destroyed)) {
         const err = new Error('Pipeline called on destroyed stream');
         for (const strm of streams) {
-            if (!strm.destroyed) strm.destroy(err);
+            if (!strm.destroyed) {
+                strm.destroy(err);
+                strm.on('error', err1 => console.log(`stream_utils: error occured on stream: `, err1));
```

Comment:
This is quite a weird behavior
I would expect the streams to listen to these events in order not to crash
Doing that here means that you won't actually be able to distinguish between streams and do different logic in case of errors